assets/scripts/segments/__tests__/EmptySegment.test.js

```diff
@@ -1,32 +1,48 @@
 /* eslint-env jest */
 import React from 'react'
-import { shallow } from 'enzyme'
-import { mountWithIntl } from '../../../../test/helpers/intl-enzyme-test-helper.js'
 import { EmptySegment } from '../EmptySegment'
 import { TILE_SIZE } from '../../segments/constants'
 import { SETTINGS_UNITS_METRIC } from '../../users/constants'
+import { renderWithReduxAndIntl } from '../../../../test/helpers/render'
 
 describe('EmptySegment', () => {
   it('renders nothing when the width is 0', () => {
-    const wrapper = shallow(<EmptySegment width={0} />)
-    expect(wrapper.html()).toEqual(null)
+    const { container } = renderWithReduxAndIntl(<EmptySegment width={0} />)
+    expect(container.firstChild).toBeNull()
   })
 
   it('renders a width, and at left position 0 by default', () => {
-    const wrapper = shallow(<EmptySegment width={12.5} units={SETTINGS_UNITS_METRIC} locale="en" />)
-    expect(wrapper.find('.segment-empty').props().style.width).toEqual(`${12.5 * TILE_SIZE}px`)
-    expect(wrapper.find('.segment-empty').props().style.left).toEqual('0px')
+    const { container } = renderWithReduxAndIntl(
+      <EmptySegment width={12.5} units={SETTINGS_UNITS_METRIC} locale="en" />
+    )
+    expect(
+      container.firstChild.classList.contains('segment-empty')
+    ).toBeTruthy()
+    expect(container.firstChild.style.width).toEqual(`${12.5 * TILE_SIZE}px`)
+    expect(container.firstChild.style.left).toEqual('0px')
   })
 
   it('renders at width and left position given', () => {
-    const wrapper = shallow(<EmptySegment width={15} left={33} units={SETTINGS_UNITS_METRIC} locale="en" />)
-    expect(wrapper.find('.segment-empty').props().style.width).toEqual(`${15 * TILE_SIZE}px`)
-    expect(wrapper.find('.segment-empty').props().style.left).toEqual(`${33 * TILE_SIZE}px`)
+    const { container } = renderWithReduxAndIntl(
+      <EmptySegment
+        width={15}
+        left={33}
+        units={SETTINGS_UNITS_METRIC}
+        locale="en"
+      />
+    )
+    expect(
+      container.firstChild.classList.contains('segment-empty')
+    ).toBeTruthy()
+    expect(container.firstChild.style.width).toEqual(`${15 * TILE_SIZE}px`)
+    expect(container.firstChild.style.left).toEqual(`${33 * TILE_SIZE}px`)
   })
 
   // TODO: unskip these tests when enzyme and react-test-render support memoized components
   it.skip('renders text content', () => {
```

Comment:
Thanks for this PR, @arrlancore!

Is it possible to unskip this test now?

Reply:
Yes, its already tested actually.